content/en/docs/reference/kubectl/overview.md

```diff
@@ -69,32 +69,32 @@ The following table includes short descriptions and the general syntax for all o
-`label`        | `kubectl label (-f FILENAME \| TYPE NAME \| TYPE/NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--overwrite] [--all] [--resource-version=version] [flags]` | Add or update the labels of one or more resources.
-`logs`        | `kubectl logs POD [-c CONTAINER] [--follow] [flags]` | Print the logs for a container in a pod.
-`patch`        | `kubectl patch (-f FILENAME \| TYPE NAME \| TYPE/NAME) --patch PATCH [flags]` | Update one or more fields of a resource by using the strategic merge patch process.
-`port-forward`    | `kubectl port-forward POD [LOCAL_PORT:]REMOTE_PORT [...[LOCAL_PORT_N:]REMOTE_PORT_N] [flags]` | Forward one or more local ports to a pod.
-`proxy`        | `kubectl proxy [--port=PORT] [--www=static-dir] [--www-prefix=prefix] [--api-prefix=prefix] [flags]` | Run a proxy to the Kubernetes API server.
-`replace`        | `kubectl replace -f FILENAME` | Replace a resource from a file or stdin.
-`rolling-update`    | `kubectl rolling-update OLD_CONTROLLER_NAME ([NEW_CONTROLLER_NAME] --image=NEW_CONTAINER_IMAGE \| -f NEW_CONTROLLER_SPEC) [flags]` | Perform a rolling update by gradually replacing the specified replication controller and its pods.
-`run`        | `kubectl run NAME --image=image [--env="key=value"] [--port=port] [--replicas=replicas] [--dry-run=bool] [--overrides=inline-json] [flags]` | Run a specified image on the cluster.
-`scale`        | `kubectl scale (-f FILENAME \| TYPE NAME \| TYPE/NAME) --replicas=COUNT [--resource-version=version] [--current-replicas=count] [flags]` | Update the size of the specified replication controller.
-`version`        | `kubectl version [--client] [flags]` | Display the Kubernetes version running on the client and server.
+`annotate`    | <code>kubectl annotate (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--overwrite] [--all] [--resource-version=version] [flags]</code> | Add or update the annotations of one or more resources.
+`api-versions`    | <code>kubectl api-versions [flags]</code> | List the API versions that are available.
+`apply`            | <code>kubectl apply -f FILENAME [flags]</code> | Apply a configuration change to a resource from a file or stdin.
+`attach`        | <code>kubectl attach POD -c CONTAINER [-i] [-t] [flags]</code> | Attach to a running container either to view the output stream or interact with the container (stdin).
+`autoscale`    | <code>kubectl autoscale (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) [--min=MINPODS] --max=MAXPODS [--cpu-percent=CPU] [flags]</code> | Automatically scale the set of pods that are managed by a replication controller.
+`cluster-info`    | <code>kubectl cluster-info [flags]</code> | Display endpoint information about the master and services in the cluster.
+`config`        | <code>kubectl config SUBCOMMAND [flags]</code> | Modifies kubeconfig files. See the individual subcommands for details.
```

Comment:
@brianpursley , Thanks for this pull request. A couple of thoughts.
- The output looks good.
- Is it possible to use back ticks for the kubectl command examples (that don't have a pipe in the command) instead of Html `code` elements?
- In the future, it is better to use separate PRs for the different localized files.

Reply:
Sure, np.  I will update to use back-ticks where possible.

Reply:
@kbhawkey ptal 

https://deploy-preview-19450--kubernetes-io-master-staging.netlify.com/docs/reference/kubectl/overview/#operations